EventsTable_Stratified.ipynb

```diff
@@ -0,0 +1,2253 @@
+{
```

Comment:
This jupyter notebook appears to be largely redundant and is not within the file structure for this repository. Please either delete this notebook, or rename it and move it to the proper directory.

Reply:
Okay, yes, deleted